src/RemoteHttpInterceptor.ts

```diff
@@ -29,23 +30,23 @@ export interface SerializedResponse {
 }
 
 export class RemoteHttpInterceptor extends BatchInterceptor<
-  [ClientRequestInterceptor, XMLHttpRequestInterceptor]
+  [ClientRequestInterceptor, XMLHttpRequestInterceptor, FetchInterceptor]
 > {
+  requestId2Response: Map<string, { request: InteractiveRequest, resolve: () => void }> = new Map();
   constructor() {
     super({
       name: 'remote-interceptor',
       interceptors: [
         new ClientRequestInterceptor(),
         new XMLHttpRequestInterceptor(),
+        new FetchInterceptor(),
```

Comment:
Happy to move this change out (if wanted) of the PR.

Reply:
Yeah, these are covered in #429 (#435). Fetch interceptor wasn't in the node preset because we used to support Node.js 16 where fetch wasn't global. That's not the case anymore so we should proceed with merging that change.